### DISCUSSION TITLE
[Welcome] 欢迎来到 WFA 论坛👋👋

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Wfa.Uwp#18] [Announcements]
## 你可以在论坛进行以下活动...

- 问关于应用或 Warframe 的问题 💭
- 讨论 💬
- 变成好奇宝宝 💡
- 反馈问题 📑
- 欢迎新成员 🙋
- 分享关于 WFA 的一切 😆

## You can do the following activities in the forum...

- Ask a question about the app or Warframe 💭
- discuss 💬
- Become a curious baby 💡
- Feedback questions 📑
- Welcome new members 🙋
- Share all about WFA 😆

---

*但请千万不要发布任何政治言论，不要涉及黄赌毒，网络不是法外之地👮*

*But please do not post any political remarks, do not involve pornography, gambling and drugs, the Internet is not a place outside the law 👮*

**Comment by lypxl20001** [comment]
你好请问会添加通知功能么？比如夜灵平原时间通知

**Comment by qunianfengxue** [comment]
最新的版本我没看到紫卡计算器和自带的市场以及登陆按钮，就是为了紫卡计算才买的。。。。。。。

**Comment by chancecao9812** [comment]
这是个好项目，最近有兴趣学.net 这个就算个很好的学习平台，pick it
